omero/users/cli/chgrp.txt

```diff
@@ -0,0 +1,140 @@
+Moving objects between groups
+-----------------------------
+
+The :omerocmd:`chgrp` command moves objects between groups. Further help is
+available using the :option:`-h` option::
+
+    $ bin/omero chgrp -h
+
+The :command:`chgrp` command will remove entire graphs of objects based on the
+IDs of the topmost objects. The command can be modified to include the movement
+of objects that would, by default, be excluded or exclude objects that would,
+by default, be included using the :option:`--include` and :option:`--exclude`
+options.
+
+Unlike the :ref:`delete command <cli_delete>`, objects of the three annotation
+types, `FileAnnotation`, `TagAnnotation` and `TermAnnotation` are not moved
+by default when the objects to which they are linked are moved.
```

Comment:
Maybe something like, "... are not *treated specially*, so they are not moved ..."